libopenage/gui/guisys/private/gui_subtree_impl.cpp

```diff
@@ -1,8 +1,10 @@
 // Copyright 2015-2017 the openage authors. See copying.md for legal info.
 
 #include "gui_subtree_impl.h"
+#include "gui_renderer_impl.h"
 
 #include <cassert>
+#include <ciso646>
```

Comment:
http://en.cppreference.com/w/cpp/header/ciso646
> ... in a conforming implementation this header is empty.
It may still be required in order to have the alternative operator representations in old or non-conformant compilers.

And you would've guessed it already, MSVC is non-conforming at least in this case.